docs/sources/tempo/configuration/grafana-alloy/automatic-logging.md

```diff
@@ -0,0 +1,116 @@
+---
+title: 'Automatic logging: Trace discovery through logs'
+description: Automatic logging provides an easy and fast way of getting trace discovery through logs.
+menuTitle: Automatic logging
+weight: 200
+aliases:
+- /docs/tempo/grafana-alloy/automatic-logging
+---
+
+# Automatic logging: Trace discovery through logs
+
+Running instrumented distributed systems is a very powerful way to gain
+understanding over a system, but it brings its own challenges. One of them is
+discovering which traces exist.
+
+Using the span logs connector, you can use Alloy to perform automatic logging.
+
+In the beginning of Tempo, querying for a trace was only possible if you knew
+the ID of the trace you were looking for. One solution was automatic logging.
+Automatic logging provides an easy and fast way of discovering trace IDs
+through log messages.
+Well-formatted log lines are written to a logs exporter
+for each span, root, or process that passes through the tracing
+pipeline. This allows for automatically building a mechanism for trace discovery.
+On top of that, you can also get metrics from traces using a logs source, and
+allow quickly jumping from a log message to the trace view in Grafana.
+
+While this approach is useful, it isn't as powerful as [TraceQL]({{< relref
+"../../traceql" >}}). If you are here because you know you want to log the
+trace ID, to enable jumping from logs to traces, then read on.
+
+If you want to query the system directly, read the [TraceQL
+documentation]({{< relref "../../traceql" >}}).
```

Comment:
```suggestion
While this approach is useful, it isn't as powerful as TraceQL. 
If you are here because you know you want to log the
trace ID, to enable jumping from logs to traces, then read on.

If you want to query the system directly, read the [TraceQL
documentation](https://grafana.com/docs/tempo/<TEMPO_VERSION>/traceql).
```